modules/scripts-utils/pom.xml

```diff
@@ -30,6 +30,7 @@
     <module>epub3-ocf-utils</module>
     <module>epub3-pub-utils</module>
     <module>epubcheck-adapter</module>
+    <module>ace-adapter</module>
```

Comment:
again a detail, but the modules are ordered alphabetically for easier manual lookup :-)